SingularityUI/app/components/common/modal/ModalWrapper.es6

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 
-export const getClickComponent = (component) => (
+export const getClickComponent = (component, doFirst = (() => Promise.resolve())) => (
```

Comment:
I initially thought so too, but when I try to do that I get `Uncaught TypeError: function resolve() { [native code] } called on non-object`.